recipes/intel_mpi.rb

```diff
@@ -39,7 +39,10 @@
   cwd node['cfncluster']['sources_dir']
   code <<-INTELMPI
     set -e
-    ./aws_impi.sh install -check_efa 0 -version #{node['cfncluster']['intelmpi']['version']}
+    tar -xf l_mpi_#{node['cfncluster']['intelmpi']['version']}.tgz
```

Comment:
can we fetch this as part of a remote_file directive and have it downloaded in the sources dir or tmp dir?
you could just update the part above that downloads the installer which is not used anymore.

Reply:
This does download into the sources directory already. Not sure what you mean.

Reply:
I think you can ignore my comment :)